include/PR/os_cont.h

```diff
@@ -43,21 +43,21 @@ typedef struct {
 
 // TODO: use real libultra button defines instead of this
 /* Buttons */
-#define BTN_CRIGHT      0x0001
-#define BTN_CLEFT       0x0002
-#define BTN_CDOWN       0x0004
-#define BTN_CUP         0x0008
-#define BTN_R           0x0010
-#define BTN_L           0x0020
-#define BTN_RESET       0x0080
-#define BTN_DRIGHT      0x0100
-#define BTN_DLEFT       0x0200
-#define BTN_DDOWN       0x0400
-#define BTN_DUP         0x0800
-#define BTN_START       0x1000
-#define BTN_Z           0x2000
-#define BTN_B           0x4000
-#define BTN_A           0x8000
+#define R_CBUTTONS      0x0001
+#define L_CBUTTONS      0x0002
+#define D_CBUTTONS      0x0004
+#define U_CBUTTONS      0x0008
+#define R_TRIG          0x0010
+#define L_TRIG          0x0020
+#define RESET_BUTTON    0x0080 /* "neutral reset": Corresponds to holding L+R and pressing S */
+#define R_JPAD          0x0100
+#define L_JPAD          0x0200
+#define D_JPAD          0x0400
+#define U_JPAD          0x0800
+#define START_BUTTON    0x1000
+#define Z_TRIG          0x2000
+#define B_BUTTON        0x4000
+#define A_BUTTON        0x8000
```

Comment:
maybe we can talk about having a game-side wrapper defines with the "good" names? 
I get wanting libultra to have original names, but we specifically named them to something better a few years ago cause the original ones are not great

Reply:
I'm not too opposed to the idea. Curious on other people's thoughts?

Reply:
I can see most people preferring the game-side wrappers instead. Personally I like libultra defines, but if most people prefer the other ones then it is fine to me
Maybe put them in `macros.h`? I can't think on a better place right now

Reply:
Went ahead and just changed the buttons back. Currently just added them to this header, but have also gone and mark places which are different to original headers.

Reply:
Would say I prefer the game-side wrappers as we had it before, but am totally fine with having wrappers to the original libultra names